gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/JavaServiceDistributionPlugin.groovy

```diff
@@ -46,7 +46,8 @@ class JavaServiceDistributionPlugin implements Plugin<Project> {
 
         project.configurations.create('goJavaLauncherBinaries')
         project.dependencies {
-            goJavaLauncherBinaries 'com.palantir.launching:go-java-launcher:1.2.0'
+            // TODO (gregakinman): update to 1.4.0 once released
```

Comment:
blocked on merge and release of https://github.com/palantir/go-java-launcher/pull/49

Reply:
tracking at https://github.com/palantir/sls-packaging/issues/308